state/redis/redis.go

```diff
@@ -403,7 +403,7 @@ func (r *StateStore) Multi(request *state.TransactionalStateRequest) error {
 
 	pipe := r.client.TxPipeline()
 	for _, o := range request.Operations {
-		//nolint:golint,nestif
+		//revive:disable,nestif
```

Comment:
Should it be two different comments? `nestif` and `revive` are separate linters.